examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -165,13 +174,14 @@
 
 # This object uses method='kmeans' for KMeans clustering with 10mm smoothing
 # and standardization ON
-kmeans = Parcellations(method='kmeans', n_parcels=50,
+start = time.time()
+kmeans = Parcellations(method='kmeans', n_parcels=2000,
```

Comment:
Why did we change to 2000 parcels? kmeans is not well suited for many parcels.

I would rather say in the text above that when we want a small number of parcels, KMeans is well suited, and then use 50 parcels here.